odxtools/comparamsubset.py

```diff
@@ -20,10 +20,10 @@
 
 @dataclass(kw_only=True)
 class ComparamSubset(OdxCategory):
-    comparams: NamedItemList[Comparam]
-    complex_comparams: NamedItemList[ComplexComparam]
-    data_object_props: NamedItemList[DataObjectProperty]
-    unit_spec: UnitSpec | None
+    comparams: NamedItemList[Comparam] = field(default_factory=NamedItemList)
+    complex_comparams: NamedItemList[ComplexComparam] = field(default_factory=NamedItemList)
+    data_object_props: NamedItemList[DataObjectProperty] = field(default_factory=NamedItemList)
+    unit_spec: UnitSpec | None = None
     category: str | None  # mandatory in ODX 2.2, but non-existent in ODX 2.0
```

Comment:
No default value here?

Reply:
thanks for the catch: these changes where done in a mostly automated manner and the comment at the end of the line seems to have screwed up my regexp. that said, for our default ODX version (2.2) this field is mandatory, so we should probably require to specify it explicitly?

Reply:
Fine, just wanted to make sure

Reply:
ok, while it probably better not to specify a default value in this specific case, I found a few other instances where this happened: [fixed](https://github.com/mercedes-benz/odxtools/pull/411/commits/cb65db5d475fcfe67948b47e763403f363031881), thanks for bringing this up!